biz_portal/apps/portal/templates/portal/business.html

```diff
@@ -1,5 +1,134 @@
+                  <summary class="mdc-expansion__summary">
+                    <span class="business-profile-location mdc-list-item">
+                      <span class="status-icon-span">
+                        <img class="status-icon" src="../../../../static/images/ellipse.svg" />
+                      </span>
+                      <span class="type-header">Status:</span>
+                      <span class="mdc-expansion__header">{{ object.status }}</span>
+                    </span>
+                    <div data-mdc-tooltip="This is how a tooltip will display when clicked" >
+                      <i class="material-icons mdc-list-item__graphic" aria-hidden="true">info</i>
+                    </div>
+                  </summary>
+                </details>
+                <details class="mdc-theme--background mdc-elevation--z2">
+                  <summary class="mdc-expansion__summary">
+                    <span class="business-profile-location mdc-list-item">
+                      <i class="material-icons mdc-list-item__graphic" aria-hidden="true">work</i>
+                      <span class="type-header">Type:</span>
+                      <span class="mdc-expansion__header">{{ object.organisation_type }}</span>
+                    </span>
+                    <div data-mdc-tooltip="This is how a tooltip will display when clicked" >
+                      <i class="material-icons mdc-list-item__graphic" aria-hidden="true">info</i>
+                    </div>
+                  </summary>
+                </details>
+                <details class="mdc-theme--background mdc-elevation--z2">
+                  <summary class="mdc-expansion__summary">
+                    <span class="business-profile-location mdc-list-item">
+                        <span class="registration-icon-span">
+                          <img class="registration-icon" src="../../../../static/images/content-paste.svg" />
+                          </span>
+                      <span class="type-header">Reg:</span>
+                      <span class="mdc-expansion__header">{{ object.registration_number }}</span>
+                    </span>
+                    <div data-mdc-tooltip="This is how a tooltip will display when clicked" >
+                      <i class="material-icons mdc-list-item__graphic" aria-hidden="true">info</i>
+                    </div>
+                  </summary>
+                </details>
+              </div>
+              <div class="business-profile-more-details">
+                <p class="more-details-text">Want to add more details?</p>
+                <button class="mdc-button more-details-button">
+                  <span class="mdc-button__label">Claim this business</span>
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </main>
+
+      <script src="{% static 'biz-portal.bundle.js' %}" defer></script>
+  </body>
```

Comment:
/body and /html should only be in base, not inside content blocik

Reply:
Covered